velox/exec/HashProbe.h

```diff
@@ -312,6 +312,20 @@ class HashProbe : public Operator {
   // memory reclamation or operator close.
   void clearBuffers();
 
+  // Returns the estimated row size of the projected output columns. nullopt
+  // will be returned if insufficient column stats is presented in 'table_', or
+  // the row size variation is too large. The row size is too large if ratio of
+  // max row size and avg row size is larger than 'toleranceRatio'.
+  std::optional<uint64_t> estimatedRowSize(
+      const std::vector<vector_size_t>& varColumnsStats,
+      uint64_t totalFixedColumnsBytes,
+      double toleranceRatio);
```

Comment:
Do we want to configure this or just make it a const ratio inside estimatedRowSize?

Reply:
I think we can leave it as a const. I don't see a strong need of making it configurable unless we run into cases in the future that needs it. Then we can always come back and make this configurable.